app/assets/stylesheets/application.css.scss

```diff
@@ -81,3 +81,14 @@ path {
  top: 100%;
  left: 0;
 }
+
+/* Add custom Font */
+@font-face {
+        font-family: "Little Flying Spiders";
+        src: url(LittleFlyingSpiders.ttf) format("truetype");
```

Comment:
Line should be indented 2 spaces, but was indented 8 spaces<br>URLs should be enclosed in quotes